src/components/Calculator.js

```diff
@@ -11,67 +11,219 @@ const CalculatorLayout = styled.div`
   box-shadow: 5px 20px 40px 1px #888888;
 `;
 
-const Calculator = () => (
-  <CalculatorLayout>
-    <Display result="0" />
-    <Button backgroundColor="gainsboro" color="black">
-      C
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      ±
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      %
-    </Button>
-    <Button backgroundColor="orange" color="white">
-      ÷
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      7
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      8
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      9
-    </Button>
-    <Button backgroundColor="orange" color="white">
-      ×
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      4
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      5
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      6
-    </Button>
-    <Button backgroundColor="orange" color="white">
-      -
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      1
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      2
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      3
-    </Button>
-    <Button backgroundColor="orange" color="white">
-      +
-    </Button>
-    <Button backgroundColor="gainsboro" size="large" color="black">
-      0
-    </Button>
-    <Button backgroundColor="gainsboro" color="black">
-      ,
-    </Button>
-    <Button backgroundColor="orange" color="white">
-      =
-    </Button>
-  </CalculatorLayout>
-);
+const Calculator = () => {
+  const [result, setResult] = useState("0");
```

Comment:
This should be a class not a function. The hooks are meant for exceptions. Also this one should be a container (I think you should have read about component and containers or smart components and dumb components).